feat/gene/gene_test.go

```diff
@@ -233,6 +242,26 @@ var transcriptTests = []struct {
 		ExonicLen: 1200,
 	},
 	{
```

Comment:
It's a test for the correct use of the feature chain semantics by these functions.